core/src/main/java/org/lflang/validation/LFValidator.java

```diff
@@ -1036,15 +1035,6 @@ public void checkState(StateVar stateVar) {
     }
   }
 
-  @Check(CheckType.FAST)
```

Comment:
This would be the location to issue a warning if `STP` is used instead of `STA`.